Kickstarter-iOS/Features/SettingsAccount/Controller/SettingsAccountViewController.swift

```diff
@@ -189,13 +189,8 @@ extension SettingsAccountViewController {
     case .createPassword:
       return CreatePasswordViewController.instantiate()
     case .changeEmail:
-      if #available(iOS 15, *) {
-        let changeEmailView = ChangeEmailView()
-
-        return UIHostingController(rootView: changeEmailView)
-      } else {
-        return ChangeEmailViewController.instantiate()
-      }
+      let changeEmailView = ChangeEmailView()
+      return UIHostingController(rootView: changeEmailView)
```

Comment:
Do we want to clean up/delete the old ChangeEmailViewController, or leave it until SwiftUI is no longer 'experimental'? The new ChangeEmailView is definitely missing some tests, for instance, but it's not like this was behind a feature flag...